driver/annotations.md

```diff
@@ -2,11 +2,11 @@
 
 # Intro
 
-Once you have written the [AST parser](driver/sdk.md) the next step is to write
+Once you have written the [AST parser](https://doc.bblf.sh/driver/sdk.html) the next step is to write
```

Comment:
docbook does parse links such as `driver/sdk.md` and transforms them to the right links on building. The problem here is that links are always relative to the current file. So if you want to link `driver/sdk.md` from the `driver/annotations.md` file, you have to use `sdk.md` or `../driver/sdk.md`, but not `driver.sdk` (which produces a link to `driver/driver/sdk.md`).